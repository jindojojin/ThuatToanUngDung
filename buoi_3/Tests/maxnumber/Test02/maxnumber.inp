45244463201253525321202221212121212120
1425212360000000000000014253625411220200000000102030
