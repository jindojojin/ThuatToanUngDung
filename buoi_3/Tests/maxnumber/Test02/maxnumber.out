6555444333222222222111111000
